aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,84 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
+- **Navigation** - Commenced research for [Ascending Navigation.](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=2382%3A13827)
+
+### Stay current with the latest changes
+- **Toast Notifications** - Updated guidance on [Toast Notifications](https://design-system.hpe.design/templates/toast-notifications?q=toast)
+
+## December 6 - December 17, 2021
+
+### "Big Things" progress updates
+Here are the latest developments regarding roadmap items:
+- **Button** 
+   - Completed guidance on placement of the Cancel Button in a Form and Modal in [Figma]( https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=667%3A385) and published on the [Design System site](https://design-system.hpe.design/components/button?q=but#styling)
+   - Updated Figma with the [new hover states for Primary and Color Buttons.](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component)
+   - Updated Button Figma file on Cancel Buttons following [default styling in Forms and Modals.](https://www.figma.com/file/Oi5UEEQ33VCAyOKQcZ8Nr8/HPE-Button-Component?node-id=4688%3A9756)
+- **Tag** 
+   - Commenced [explorations for Tag component.](https://www.figma.com/file/8zrInotU4JK9LWP4WyZy25/HPE-Design-System-Tags?node-id=2%3A3)
+   - Completed [ideations for Tags component.](https://www.figma.com/file/8zrInotU4JK9LWP4WyZy25/HPE-Design-System-Tags?node-id=286%3A2375)
+- **Navigation** 
+   - Commenced [ideation for Matrix Navigation.](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1647%3A9780)
+   - Commenced [ideation for Reverse Navigation.](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=2486%3A14490)
+   - Provided guidance and examples on Sidebar for Side-to-Side Navigation. [Figma](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1647%3A9562) [Design System site](https://design-system.hpe.design/templates/side-to-side-navigation?q=side)
+   - Finalized ideating and documentation for Matrix Navigation. [Figma](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1647%3A9780) and published on [Design System site](https://design-system.hpe.design/templates/matrix-navigation)
+   - Finalized ideation for Global Header within [Side-to-Side Navigation.](https://www.figma.com/file/b6hiokanRCwNnc93PpHdgQ/HPE-Design-System-Navigation?node-id=1647%3A9562)
+- **Page Layouts** 
+   - Commenced an overview of [Page Layouts that provides definitions and an anatomy.](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=1876%3A16251)
```

Comment:
```suggestion
   - Started an overview of [Page Layouts that provides definitions and an anatomy](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/Page-Layouts?node-id=1876%3A16251).
```
"Commenced" isn't the friendliest/most conversational term. Maybe we can use a variety of "Started", "Began", etc.